packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -938,6 +938,20 @@ const defaultConfigs = {
     uniswapAddress: "0x360acfeb5c1548bad3583c559a646d803077236d",
     twapLength: 7200,
     invertPrice: false
+  },
+  USDXIO: {
+    type: "expression",
+    expression: "XIOETH * USDETH",
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    twapLength: 3600,
+    customFeeds: {
+      XIOETH: {
```

Comment:
This is actually the ETHXIO price.

```suggestion
      ETHXIO: {
```